dynatrace_topology/README.md

```diff
@@ -0,0 +1,41 @@
+# Agent Check: Dynatrace_topology
+
+## Overview
+
+This check monitors [Dynatrace_topology][1] through the StackState Agent.
+
+## Setup
+
+### Installation
+
+The Dynatrace_topology check is included in the [StackState Agent][2] package, so you do not
+need to install anything else on your server.
+
+### Configuration
+
+1. Edit the `dynatrace_topology.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your dynatrace_topology performance data.
+   See the [sample dynatrace_topology.d/conf.yaml][2] for all available configuration options.
+
+2. Restart the Agent
+
+## Data Collected
+
+### Metrics
+
+Dynatrace_topology does not include any metrics.
+
+### Service Checks
+
+Dynatrace_topology does not include any service checks.
+
+### Events
+
+Dynatrace_topology does not include any events.
+
+### Topology
+
+Dynatrace_topology does not include any topology.
```

Comment:
It collects topology. add what we collect here. Or check with @gggina what should we fill?

Reply:
Done. There is a task for fixing docs. It's still not done, but its not in the scope of this review.
https://stackstate.atlassian.net/browse/STAC-14108